doajtest/unit/test_crosswalks.py

```diff
@@ -117,5 +117,5 @@ def test_06_crossref_article_xml_xwalk(self):
         assert bibjson.get_one_identifier(bibjson.E_ISSN) == "9876-5432", "expected '9876-5432', received: {}".format(bibjson.get_one_identifier(bibjson.E_ISSN))
         assert bibjson.year == "2004", "expected '2004', received: {}".format(bibjson.year)
         assert bibjson.title == "Article 12292005 9:32", "expected 'Article 12292005 9:32', received: {}".format(bibjson.title)
-        assert bibjson.author == [{'name': 'Surname, Bob'}], "expected [{{'name': 'Surname, Bob'}}]', received: {}".format(bibjson.author)
+        assert bibjson.author == [{'name': 'Surname Bob'}], "expected [{{'name': 'Surname Bob'}}]', received: {}".format(bibjson.author)
```

Comment:
is this the correct representation of an author's name? "Domańska Aga" rather than "Aga Domańska"?

If we're keeping it surname then given name, the comma would be typical. But I think since our metadata form has one name field and lots of the articles I've looked at have it entered as typically western ordering, Firstname Surname, we may wish to be consistent.

I may have missed a prior discussion about this when the crossref work was originally done, though.

Reply:
I assumed surname first as it is the required piece of data, given name is optional. I think the ordering depends a lot on the country though ;) I am not sure whether we discussed that or not, that is true. @dommitchell I think it is your decision here to make.

Reply:
We should display them as First name, Last name as we do for the rest of DOAJ article metadata